pages/create-expense.js

```diff
@@ -307,7 +312,11 @@ class CreateExpensePage extends React.Component {
     const payoutProfiles = getPayoutProfiles(loggedInAccount);
 
     return (
-      <Page collective={collective} {...this.getPageMetaData(collective)}>
+      <Page
+        collective={collective}
+        {...this.getPageMetaData(collective)}
+        canonicalURL={`${getCollectivePageCanonicalURL(collective)}/expenses/new`}
```

Comment:
Sounds like `canonicalURL` is part of page metadata. Could we move this to `getPageMetaData`?

Reply:
Moved. Thanks. I missed this. 🤔